### DISCUSSION TITLE
Where can I download models created by older versions of PKsim in Model Library

### DISCUSSION THREAD

**Discussion by andanteyang** [Open-Systems-Pharmacology/Forum#779] [Questions & Problems] [answered]
By some unknown reasons the updated PKsim version (v9.1.3) can not be installed on my PC (with the error message: 0x800713ec - Asia). And using a PKsim 8 the model file in the library can not be opened. Is there is way that I can download the previous models created by an older version?

Thanks in advance!

**Comment by msevestre** [comment, marked as answer]
@andanteyang Based on some quick research, it seems that you will need to install the .NET Framework first

Please follow the following link
https://go.microsoft.com/fwlink/?LinkID=863265

This should be installed with the setup automatically but who knows
Also please make sure you have  admin rights on your system before installing it

Then proceed (with admin rights obviously) to the normal installation of PKSim 
Quick question: how do you install PK-Sim? You are using the setup version right ?
https://github.com/Open-Systems-Pharmacology/Suite/releases/download/v9.1/OSPSuite-Full.9.1.3.exe
